libmscore/instrument.cpp

```diff
@@ -212,6 +215,11 @@ void Instrument::write(Xml& xml, Part* part) const
             xml.tag("useDrumset", _useDrumset);
             _drumset->save(xml);
             }
+      if (_useExpression)
+            xml.tag("dynamics", "expression");
+      else
+            xml.tag("dynamics", "velocity");
+      xml.tag("fixedVelocity", _fixedVelocity);
```

Comment:
I believe these 5 lines are the cause for the mtest failures. If those tags were written only if not set to default, this would preven the mtest failures I guess, and propbably improve compatibility